### ISSUE TITLE
[Busy indicator | Accessibility]: Focusable item with no role

### ISSUE THREAD

**Issue by PlutaKatarzyna** [SAP/ui5-webcomponents#10846]
### Bug Description

Access Assistant tool report issue of ui5-busy-indicator, that focusable element does not have a role.

In the component, there is not visible `span[data-ui5-focus-redirect]` element, that has tabindex=0. It does not have role attribute set, which trigger an issue in accessibility tool.

<img width="1668" alt="Image" src="https://github.com/user-attachments/assets/82f2c7d5-5311-41af-9cd1-108abd3fb15b" />
<img width="1668" alt="Image" src="https://github.com/user-attachments/assets/1fe765c6-ff4f-4ca4-9c7d-57fdb6815e5d" />

### Affected Component

ui5-busy-indicator

### Expected Behaviour

No issues in Busy indicator in Access Assistant

### Isolated Example

https://pages.github.tools.sap/sapudex/digital-design-system/?path=/story/components-actions-busy-indicator--over-text&globals=theme:Horizon+Brand

### Steps to Reproduce

1. Install Access Assistant Chrome extension
2. Run on Busy indicator example
3. Check "Ensure custom controls provide proper textual name, role, and state information" issue

Note: neither SiteImprove or Access Assistant doesn't check UI5 docs iframe content.

### Priority

Medium

### UI5 Web Components Version

1.24.15

### Browser

Chrome

### Additional Context

UDEx ticket reported https://github.tools.sap/sapudex/digital-design-system/issues/1107

In UDEx Library, we don't use component wrapper of UI5 Busy Indicator. We use this component directly.

### Organization

SAP UDEx Components & Tokens

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by dobrinyonkov** [comment]
Hello @SAP/ui5-webcomponents-topic-rd,

Accessibility tools are reporting that the data-ui5-focus-redirect element lacks semantic meaning but is still considered interactive.

Could you please investigate this issue? You might need to exclude this element from accessibility mappings. However, aria-hidden doesn't seem to work—consider using role="presentation" instead.

Kind regards,
Dobrin
